stripe/src/main/java/com/stripe/android/googlepay/StripeGooglePayContract.kt

```diff
@@ -128,3 +129,31 @@ internal class StripeGooglePayContract :
         }
     }
 }
+
+fun Status.getErrorMessage(): String? {
```

Comment:
Once we agree on the strings, I will move them to resources to localize them in a future version.